lib/spack/docs/tutorial_modules.rst

```diff
@@ -103,56 +365,66 @@ Next you should install a few modules that will be used in the tutorial:
    $ spack install netlib-scalapack ^mpich ^netlib-lapack
    $ spack install py-scipy ^openblas
 
-In the end your environment should look something like:
+
+.. _module_file_tutorial_non_hierarchical:
+
+-----------------------------
+Non-hierarchical module files
+-----------------------------
+
+If you arrived to this point you should have an environment that looks similar to:
 
 .. code-block:: console
 
-  $ module avail
+  root@module-file-tutorial:/# module av
```

Comment:
I think it's best to avoid shortcuts and abbreviations. I like `module avail` better.